v86.css

```diff
@@ -305,3 +305,24 @@ h4 {
     z-index: 2;
     position: relative;
 }
+
+.blink {
+    animation: blink 0.6s step-start infinite;
+}
+
+.cursor {
+    animation: cursor 0.2s step-start infinite;
```

Comment:
I don't understand this code. Is the cursor background colour somehow animated?

Reply:
yes, 0.1s background is as it is, 0.1s background is transparent (cursor is not visible), then all repeats.

Reply:
Makes sense. FYI, I'll change this to 0.6, because 0.2 is too fast for my taste, and 0.6 roughly matches qemu's cursor blinking speed.